### PR TITLE
Add links to create org flow behind flag

### PR DESCRIPTION
**Pull request by bduffany**

### CODE REVIEW

File: app/menu/menu.tsx [hunk 1]
```diff
@@ -94,7 +94,14 @@ export default class MenuComponent extends React.Component {
                     !this.props.user?.selectedGroup.ownedDomain &&
                     !this.props.user?.isInDefaultGroup() && (
                       <li onClick={this.dismissMenu.bind(this)}>
-                        <a target="_blank" href="https://buildbuddy.typeform.com/to/PFjD5A">
+                        <a
```
Comment (siggisim) [inline app/menu/menu.tsx:97]: i'd add a little `router.navigateToCreateOrg()` helper method in router.tsx like this: https://github.com/buildbuddy-io/buildbuddy/blob/master/app/router/router.tsx#L43

navigating using `<a href=` causes a full page reload, whereas navigating with the router just causes an react re-render

you can have router check capabilities and send you to typeform if necessary or keep that logic here

File: app/router/router.tsx [hunk 2]
```diff
@@ -153,6 +153,8 @@ export class Path {
   static commitHistoryPath = "/history/commit/";
   static setupPath = "/docs/setup/";
   static settingsPath = "/settings/";
+  static createOrgPath = "/org/create";
```
Comment (siggisim) [inline app/router/router.tsx:156]: it's gnarly and totally my fault, but you'll need to add `/org/` here so the server knows that's a path on which to serve the app (if you do a full page refresh)

https://github.com/buildbuddy-io/buildbuddy/blob/b10b73c12757eee49b990e619a997857a16726ed/server/libmain/libmain.go#L247

**Review by siggisim — CHANGES_REQUESTED** [review]

File: app/router/router.tsx (hunk 2 above)
Comment (bduffany) [reply app/router/router.tsx:156]: Done -- thanks!

File: app/menu/menu.tsx (hunk 1 above)
Comment (bduffany) [reply app/menu/menu.tsx:97]: Ah kk, what do you think about adding a `Link` component which automatically does the right thing?

Comment (siggisim) [reply app/menu/menu.tsx:97]: let's stick with the router methods for now, we can migrate all router usage in the future if that seems better - but right now we have some logic in there to check if the app is capable to navigating to the destination etc that would make a link component kinda messy

Comment (bduffany) [reply app/menu/menu.tsx:104]: Ah I see -- Done! Wasn't sure whether to show the alert in `Router`, LMK if it makes sense

File: app/router/router.tsx [hunk 3]
```diff
@@ -100,6 +100,16 @@ class Router {
     this.navigateTo(Path.commitHistoryPath + commit);
   }
 
+  navigateToCreateOrg() {
+    if (!capabilities.createOrg) {
+      alert(
```
Comment (siggisim) [inline app/router/router.tsx:105]: what if we do this here:
```
window.open('https://buildbuddy.typeform.com/to/PFjD5A', "_blank") 
```

then the call site can be super clean with no need for preventDefault, path, <a href=, target, importing Path, etc

just a ```<div className="clickable" onClick={() => router. navigateToCreateOrg()}>Create organization</div>```

Comment (bduffany) [reply app/router/router.tsx:105]: Done

Comment (bduffany) [reply app/router/router.tsx:105]: FYI, the `.clickable` `div` is not keyboard accessible (unlike `<a>`) -- we might want to fix that in a clean sweep in the future.

**Review by siggisim — APPROVED** [review]
lgtm!

**Pull request merged**